frameworks/g-cloud-9/search_mapping.json

```diff
@@ -1,4 +1,27 @@
 {
+  "settings": {
+    "index": {
+      "number_of_shards": 1
```

Comment:
Does dropping to one shard have any implications for data recovery if we lose a node? It makes sense that if we have the same 1 shard duplicated across three nodes it should be fine. I just wonder what the reasoning behind having 5 shards initially was?